code/modules/mob/living/simple_animal/bot/ed209bot.dm

```diff
@@ -1,430 +1,72 @@
-/mob/living/simple_animal/bot/ed209
+/mob/living/simple_animal/bot/secbot/ed209
 	name = "\improper ED-209 Security Robot"
-	desc = "A security robot.  He looks less than thrilled."
-	icon = 'icons/mob/aibots.dmi'
-	icon_state = "ed2090"
+	desc = "A security robot. He looks less than thrilled."
+	icon_state = "ed209"
 	density = TRUE
-	anchored = FALSE
 	health = 100
 	maxHealth = 100
-	damage_coeff = list(BRUTE = 0.5, BURN = 0.7, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
 	obj_damage = 60
 	environment_smash = ENVIRONMENT_SMASH_WALLS //Walls can't stop THE LAW
 	mob_size = MOB_SIZE_LARGE
 
-	radio_key = /obj/item/encryptionkey/headset_sec
-	radio_channel = RADIO_CHANNEL_SECURITY
-	bot_type = SEC_BOT
 	model = "ED-209"
-	bot_core_type = /obj/machinery/bot_core/secbot
 	window_id = "autoed209"
 	window_name = "Automatic Security Unit v2.6"
-	allow_pai = 0
-	data_hud_type = DATA_HUD_SECURITY_ADVANCED
-	path_image_color = "#FF0000"
-	carryable = FALSE
-
+	ranged = TRUE
 	var/lastfired = 0
 	var/shot_delay = 15
-	var/lasercolor = ""
-	var/disabled = FALSE //A holder for if it needs to be disabled, if true it will not seach for targets, shoot at targets, or move, currently only used for lasertag
-
-
-	var/mob/living/carbon/target
-	var/oldtarget_name
-	var/threatlevel = 0
-	var/target_lastloc //Loc of target when arrested.
-	var/last_found //There's a delay
-	var/declare_arrests = TRUE //When making an arrest, should it notify everyone wearing sechuds?
-	var/idcheck = TRUE //If true, arrest people with no IDs
-	var/weaponscheck = TRUE //If true, arrest people for weapons if they don't have access
-	var/check_records = TRUE //Does it check security records?
-	var/arrest_type = FALSE //If true, don't handcuff
-	var/projectile = /obj/projectile/energy/electrode //Holder for projectile type
-	var/shoot_sound = 'sound/weapons/taser2.ogg'
-	var/cell_type = /obj/item/stock_parts/cell
-	var/vest_type = /obj/item/clothing/suit/armor/vest
```

Comment:
Why did these just get removed instead of storing what was used to make the bot? The components used in construction should be stored so they can be dropped via `drop_part()` when the bot is destroyed. 

I'd recommend storing them in a list rather than individual variables though if you were trying to keep it clean, I touch more on this later in the explode() proc review.

Reply:
handled for now. Can confirm they drop during /explode()

Reply:
This is not handled, the variables are still removed instead of keeping the items stored for dropping later. 

Again, a list is the easiest way to do this.